devices/common_patches/usb_otg/hs_fixes_446_469.yaml

```diff
@@ -0,0 +1,15 @@
+OTG_HS_GLOBAL:
+  GCCFG:
+    _delete:
+      - DCDET
+      - PDET
+      - SDET
+      - PS2DET
+      - BCDEN
+      - DCDEN
+      - PDEN
+      - SDEN
+
+  _modify:
+    CID:
+      resetValue: 0x2100
```

Comment:
![image](https://user-images.githubusercontent.com/7845120/215286377-d94167bc-8989-4996-8e28-6d3ecb69c8f2.png)

Isn't this `0x1200`?

Reply:
I see this in RM0386 Rev 5 and RM0390 Rev 6:
![Screenshot_20230129_121419](https://user-images.githubusercontent.com/3072754/215316781-dfdb70c1-8b2d-4498-95dc-9ba4839a86d4.png)

Reply:
My bad, I was looking at the F4x5 RM (RM0090 Rev 19), not the F446/F469 RM.